Mathlib/Data/List/Basic.lean

```diff
@@ -392,22 +392,20 @@ theorem cons_eq_append_iff {a b c : List α} {x : α} :
 
 #align list.append_inj_left' List.append_inj_left'ₓ -- implicits order
 
-theorem append_left_cancel {s t₁ t₂ : List α} (h : s ++ t₁ = s ++ t₂) : t₁ = t₂ :=
-  (append_right_inj _).1 h
-#align list.append_left_cancel List.append_left_cancel
+@[deprecated] alias append_left_cancel := append_cancel_left
```

Comment:
please add a comment: `-- deprecated since 2024-01-18` to both aliases

Reply:
I added the comments to these aliases.